auth_v3.go

```diff
@@ -0,0 +1,206 @@
+package swift
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"strings"
+)
+
+const (
+	AUTH_METHOD_TOKEN         = "token"
```

Comment:
The aren't very go-like these constants....  Do they need to be public?  if not can you rename them `authMethodToken` etc